frontend/src/studentPage/Chat.tsx

```diff
@@ -59,6 +60,24 @@ export default function Chat(props: Readonly<Props>){
             console.error("Error while saving or fetching messages:", error);
         }
     }
+    const handleDelete = async (lessonId: string, message: Message) => {
+        if (lessonId) {
+            await lessonService.deleteMessage(lessonId, message);
+            const updatedLessons = updateLessons(lessonId, message);
+            setLessons(updatedLessons);
+            const response = await lessonService.getAllMessageByLessonId(lessonId);
+            setMessages(response.data);
+            setDescription('');
+        }
+    }
+    const updateLessons = (lessonId: string, message: Message) => {
```

Comment:
The naming is a bit missleading. Maybe something like `deleteMessageFromLessons` would be better.